lib/manageiq/api/common/user.rb

```diff
@@ -4,36 +4,19 @@ module Common
       class User
         IDENTITY_KEY = 'x-rh-identity'.freeze
 
-        def username
-          validate_and_return(__method__)
-        end
-
-        def email
-          validate_and_return(__method__)
-        end
-
-        def first_name
-          validate_and_return(__method__)
-        end
-
-        def last_name
-          validate_and_return(__method__)
-        end
-
-        def is_active?
-          validate_and_return(:is_active)
-        end
-
-        def is_org_admin?
-          validate_and_return(:is_org_admin)
-        end
-
-        def is_internal?
-          validate_and_return(:is_internal)
-        end
-
-        def locale
-          validate_and_return(__method__)
+        %w[
+          username
```

Comment:
Would it make sense to move this list into a constant?

Reply:
I don't believe so, because it won't be used any other place than here, and putting it in a constant a) exposes it outside of this class and b) doesn't allow it to be GC'd